### DISCUSSION TITLE
August 15th 2024 Community Call

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#24123] [Community Call]
Agenda 

- PowerShell releases  @SydneyhSmith
- PS 7.5 preview 4 env var @JamesWTruher 
- MSI Telemetry @SteveL-MSFT 
- DSC v3 update @SteveL-MSFT 
- PowerShell Gallery statistics @alerickson 
- Techmentor @SydneyhSmith 
- PowerShell Saturday
- MiniCon
- Community demos

Please post any topics/demos/questions

**Comment by Gijsreyn** [comment]
@SydneyhSmith Hey Sydney, can I give a small demo related to https://github.com/PowerShell/DSC/pull/507?

**Reply by SydneyhSmith** [reply]
Thant would be awesome-- let's plan on it!

**Reply by Gijsreyn** [reply]
It's a small one, shouldn't take more than 5 minutes :)

**Comment by ThomasNieto** [comment]
I also have a demo for AnyPackage command not found feedback provider and command predictor.

**Reply by SydneyhSmith** [reply]
Thanks that sounds great

**Reply by ThomasNieto** [reply]
Here are some links from the demo and currently supported package providers.

```powerhsell
Install-PSResource AnyPackage.PSResourceGet
Install-PSResource AnyPackage.Scoop

# Import module to make package provider available
Import-Module AnyPackage.PSResourceGet
Import-Module AnyPackage.Scoop
```

[Github](https://github.com/anypackage)
[Website](https://anypackage.dev)

![image](https://github.com/user-attachments/assets/840a6948-26cf-4010-bd07-a9da45ae2fd4)

**Comment by Gijsreyn** [comment]
For anyone interested in the PowerShell module for DSC V3 and the discussion:

[[WIP] Microsoft.PowerShell.DSC module](https://github.com/PowerShell/DSC/pull/507)
[PowerShell module for DSCv3](https://github.com/PowerShell/DSC/discussions/508)
